src/lib/Components/Browse.php

```diff
@@ -30,7 +30,7 @@ public function renderToString()
         $selected = $this->getLocationId();
         // should be rendered with a twig template
         return '<ez-browse
-            class="ez-button"
+            class="ez-button  ez-button-action"
```

Comment:
This button is in the discovery bar (left menu). I am not sure it should be in this PR.
But we can keep this change if you want :)